fluent/test/parser_behavior_test.js

```diff
@@ -20,20 +19,40 @@ readdir(fixtures, function(err, filenames) {
     filename => filename.endsWith('.ftl')
   );
 
-  suite.skip('Behavior tests', function() {
-    for (const filename of ftlnames) {
-      const filepath = join(fixtures, filename);
-      test(filename, function() {
-        return readfile(filepath).then(file => {
-          const { directives, source } = preprocess(file);
-          const expected = directives.join('\n') + '\n';
-          const [entries, annots] = parse(source);
-          const actual = annots.map(serializeAnnotation).join('\n') + '\n';
-          assert.deepEqual(
-            actual, expected,
-            'Actual Annotations don\'t match the expected ones'
-          );
-        });
+  suite('Behavior tests', function() {
+    for (const ftlfilename of ftlnames) {
+      const jsonfilename = ftlfilename.replace(/ftl$/, 'json');
+      const ftlpath = join(ftlFixtures, ftlfilename);
+      const jsonpath = join(jsonFixtures, jsonfilename);
+
+      test(ftlfilename, async function() {
+        const [source, json] = await Promise.all([ftlpath, jsonpath].map(readfile));
+
+        const [entries] = parse(source);
+        const expectedEntries = JSON.parse(json);
+
+        for (const [id, expected] of Object.entries(expectedEntries)) {
```

Comment:
Iterating over `expectedEntries` here means that the runtime parser could possibly parse _more_ entries and we wouldn't learn about it from these tests. I'd like to fix this in a follow-up later on.